packages/vanilla/src/helpers/css/mixins/typography.scss

```diff
@@ -1,19 +1,38 @@
+@import '~basics/colors/colors.scss';
 // Typography Base
 // Give an element standard typographic properties
 // Usage Example:
 // .selector {
 //     @include typography-base;
 // }
 
+$typography-base-color: color('hig-gray-70');
```

Comment:
Main changes:

* Export the base color to allow custom color opacity
* Provide large/medium/small font size variations as mixins